cdp_scrapers/instances/portland.py

```diff
@@ -239,13 +319,35 @@ def get_votes(self, event_page: BeautifulSoup) -> Optional[List[Vote]]:
         # Clearly “Yea” -> VoteDecision.APPROVE, but we don’t have enough information
         # for what words to map to other constants.
 
-        return reduced_list(
-            [
-                self.get_none_if_empty(
-                    Vote(decision=None, person=self.get_person(None))
-                ),
-            ],
+        vote_element_uncle = event_page.find(
```

Comment:
Again here Shak, I believe we need to change the method signature, like for `get_matter()` since you are now passing in a `Tag` for a particular event minutes item, not `BeautifulSoup` of the entire web page.